### DISCUSSION TITLE
Sending email when power on the board.

### DISCUSSION THREAD

**Discussion by imoldovavan** [arendst/Tasmota#21003] [General]
Can I use tasmota to send an smtp email when the board is powered on?? I want to use this with [https://github.com/Xinyuan-LilyGO/LilyGo-T-OI-PLUS/tree/main](url). I know I have to compile it myself with this function enabled, just wandering if it was possible.
Thank you,.

**Comment by sfromis** [comment]
Yes, with a suitable build, you can use Berry code or rules to invoke the `Sendmail` command after Wifi connection. Obviously not if the power on did not lead to a successful network connection.
